### PR TITLE
QSP-56 Remove Deprecated Kafka JSONPB dependency

### PR DESCRIPTION
**Pull request by rauljordan**
Part of #6327, we were using an outdated jsonpb dependency for kafka that is now deprecated. The new recommendation is to use: google.golang.org/protobuf/encoding/protojson.

### CODE REVIEW

**Comment by prestonvanloon** [comment]
This PR doesn't build. Please fix

File: beacon-chain/db/kafka/export_wrapper.go [hunk 1]
```diff
@@ -3,10 +3,8 @@
 package kafka
 
 import (
-	"bytes"
 	"context"
 
-	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
```
Comment (prestonvanloon) [inline beacon-chain/db/kafka/export_wrapper.go:8]: Did you mean to update this?

**Comment by nisdas** [comment]
Tests fail @rauljordan

**Comment by rauljordan** [comment]
Unfortunately, this PR fails to build due to a bazel dependency for the rest of our protobuf toolchain that cannot be updated easily. This is not a high priority issue, so closing until a further date

**Pull request closed without merging**